### ISSUE TITLE
Forge 1.19.2 Issue with Origins mod, estrogen re-enables fall damage for feline origin

### ISSUE THREAD

**Issue by Gwen-Hemoxia** [MayaqqDev/Estrogen#112]
Title says it all. As a feline origin in the Origins mod, you gain native total immunity to fall damage but once under the effect of Girl Power, you take fall damage again, even when not dashing.
I don't know on which mod's side the issue could be so I will report it on both sides.

This is a tragedy for us trans catgirls

**Comment by MayaqqDev** [comment]
Issue on their part, reported A LONG time ago, not my fault.

**Comment by nebulazorua** [comment]
Been having the same issue on Fabric on 1.20

It's weird, since fall_invulnerability checks the tag for is_fall, so it should be fine?

**Comment by nebulazorua** [comment]
> Been having the same issue on Fabric on 1.20
> 
> It's weird, since fall_invulnerability checks the tag for is_fall, so it should be fine?

OKAY so its because the 1.20.1 version doesn't have the change to the fall_immunity power where it uses tags, instead using the 'fall' name
its fixed in 1.20.2+ but you can use a datapack to fix it in 1.20.1 (and I think below?)
data/origins/powers/fall_immunity.json:
```json
{
    "type": "origins:invulnerability",
    "damage_condition": {
        "type": "origins:in_tag",
        "tag": "minecraft:is_fall"
    }
}
```

**Comment by Gwen-Hemoxia** [comment]
> Issue on their part, reported A LONG time ago, not my fault.

Oh, apologies, then! I tried looking but didn't find a ticket for it.
Have a nice day!
